ingestor/utils.js

```diff
@@ -0,0 +1,66 @@
+// TODO
+// await use axios post seedData fn to push data to db
+// done: delete file directory(or add to gitignore so zip files doesn’t push to Github) + drain
+
+const fs = require('fs');
+const path = require('path');
+const unzipper = require('unzipper'); // npm lib used to unzip files
+const { createReadStream } = require('fs');
+
+// async fs reads and unzip files to a directory
+
+const folderName = 'Archive.zip';
+const pathToExtract = 'unzippedArchive';
+
+const unzipFolder = (folderName, pathToExtract) => {
```

Comment:
Can the parameter name be different from the variable you declared above, I feel it's going to get confusing.

Reply:
Oh yea! Of course.